source/agora/consensus/data/Block.d

```diff
@@ -128,15 +128,10 @@ unittest
 
     The block which contains the block header and its body (the transactions).
 
-    In the current preliminary design a block contains a single transaction.
-
 *******************************************************************************/
 
 public struct Block
 {
-    /// number of transactions that constitutes a block
-    public enum TxsInBlock = 8;
```

Comment:
Perhaps just make it a `version (unittest)` ? At the moment there's a bunch of changes in the test suite which are of little relevance.

Reply:
Well I thought future tests should stop making these assumptions so I removed the constant.

Reply:
You're right, but changing an easily-grepped-for constant for `8` isn't removing the assumption

Reply:
Fair enough.